go/pkg/repeat/every_test.go

```
@@ -0,0 +1,404 @@
+	m.mu.Lock()
+	defer m.mu.Unlock()
+	m.messages = append(m.messages, logMessage{level: level, msg: msg, args: args})
+}
+
+func (m *mockLogger) getMessages() []logMessage {
+	m.mu.Lock()
+	defer m.mu.Unlock()
+	return append([]logMessage(nil), m.messages...)
+}
+
+func (m *mockLogger) reset() {
```

golangci-lint (2.2.2)

func (*mockLogger).reset is unused

(unused)